core/api/src/domain/wallet-invoices/index.types.d.ts

```diff
@@ -74,7 +74,7 @@ type WIBWithAmountState = WIBWithExpirationState & {
 }
 
 type LnAndWalletInvoice = {
-  walletInvoice: WalletInvoice
+  walletInvoice: WalletInvoice & { paymentRequest: EncodedPaymentRequest }
```

Comment:
This is temporary right? Seems like we would like `paymentRequest` to permanently be part of `WalletInvoice`?

Reply:
Yes it is temporary. Right now we want it to be required when we persist it but optional when we read it.